_posts/2020-09-24-smallrye-config-1-9.adoc

```diff
@@ -0,0 +1,206 @@
+= SmallRye Config 1.9 Released
+:page-layout: post
+:page-title: SmallRye Config 1.9 Released
+:page-synopsis: SmallRye Config 1.8 Released
+:page-tags: [microprofile,smallrye,config]
+:page-date: 2020-09-24 13:00:00.000 +0100
+:page-author: radcortez
+
+In SmallRye Config 1.9 we continue our innovation and experimentation model, by bringing additional features and
+enhancements to the API. This release introduces the following big changes:
+
+== Config Mappings
+
+A Config Mapping provides a way to group multiple configuration properties into a single cohesive API. A Config Mapping
+requires a simple interface with minimal metadata configuration.
+
+[source,java]
+----
+@ConfigMapping(prefix = "server")
+interface Server {
+    String host();
+
+    int port();
+}
+----
```

Comment:
Isn't something like this present in MP Config 2.0? Perhaps mention that?

Reply:
Sure!